Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -161,86 +178,96 @@ public extension Sequence {
         return (matching, nonMatching)
     }
 
-    /// SwifterSwift: Return a sorted array based on a key path and a compare function.
+    /// SwifterSwift: Return a sorted array based on a map function and a compare function.
     ///
-    /// - Parameter keyPath: Key path to sort by.
+    /// - Parameter map: Function that defines the property to sort by.
     /// - Parameter compare: Comparison function that will determine the ordering.
     /// - Returns: The sorted array.
-    func sorted<T>(by keyPath: KeyPath<Element, T>, with compare: (T, T) -> Bool) -> [Element] {
-        return sorted { compare($0[keyPath: keyPath], $1[keyPath: keyPath]) }
+    func sorted<T>(by map: (Element) -> T, with compare: (T, T) -> Bool) -> [Element] {
```

Comment:
This is a breaking change as you're changing the API of these functions. Even though direct usages should remain usable, any function that accepts a `KeyPath` and then passes it on to this function will fail to compile.
```swift
func mySorted<T>(by keyPath: KeyPath<Element, T>, with compare: (T, T) -> Bool) -> [Element] {
    return sorted(by: keyPath, with: compare) // error, as the `by` parameter type has changed
}
```
You can just add your version alongside it, to avoid the breaking change.